templates/account.html

```diff
@@ -30,14 +32,15 @@ <h5>{{game_name}}</h5>
     {% endfor %}
     {% endif %}
     {% endwith %}
+
     <br>
     <br>
-    <form method="POST" action="/account">
```

Comment:
Good job catching this error so that users will now be able to see their saved games